resources/migrations/20190301085026-example.edn

```diff
@@ -1,3 +0,0 @@
-{:ns rems.migrations.example
- :up-fn migrate-up
- :down-fn nil}
```

Comment:
Why was this removed? Do code based migrations not work with mount-lite?

Reply:
Simply because it's not required either.